presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -150,14 +151,15 @@ private void updateScreen(WarningsPrinter warningsPrinter)
         printQueryInfo(client.currentStatusInfo(), warningsPrinter);
     }
 
-    private String autoFormatMetricValue(String name, long value)
+    private String autoFormatMetricValue(RuntimeUnit unit, long value)
     {
-        // TODO: The current implementation of detecting metric type is hacky. An better solution is to add an enum to RuntimeMetric to track its type. But since there are no
-        // other use cases, it is too heavy to implement. We can revisit this when there are new use cases.
-        if (name.contains("Nanos")) {
+        if (unit == RuntimeUnit.NANO) {
             return formatTime(Duration.succinctNanos(value));
         }
-        return formatCount(value);
+        if (unit == RuntimeUnit.BYTE) {
```

Comment:
static import `NANO` and `BYTE`

Reply:
There is already defined a static import in this class with name BYTE, what should we do in this case?

> import static io.airlift.units.DataSize.Unit.BYTE;

Reply:
In this case we can keep it.

Reply:
sure, ill leave it as is then